diff --git a/src/Makefile.in b/src/Makefile.in
index fd79534..c576fd2 100644
--- a/src/Makefile.in
+++ b/src/Makefile.in
@@ -1333,7 +1333,7 @@ uninstall-am: uninstall-binPROGRAMS uninstall-binSCRIPTS \
 
 mkheader$(EXEEXT_FOR_BUILD): mkheader.c Makefile
 	$(CC_FOR_BUILD) $(CFLAGS_FOR_BUILD) $(CPPFLAGS_FOR_BUILD) \
-          $(LDFLAGS_FOR_BUILD) -I. -I$(srcdir) -o $@ $(srcdir)/mkheader.c
+          $(LDFLAGS_FOR_BUILD) -I. -I$(srcdir) -o $@ $(srcdir)/mkheader.c $(LIBS)
 
 assuan.h: assuan.h.in mkheader$(EXEEXT_FOR_BUILD) $(parts_of_assuan_h)
 	./mkheader$(EXEEXT_FOR_BUILD) $(host_os) $(srcdir)/assuan.h.in \
