diff --git a/src/Makefile.in b/src/Makefile.in
index 66aace0..1ee3358 100644
--- a/src/Makefile.in
+++ b/src/Makefile.in
@@ -1331,7 +1331,7 @@ uninstall-am: uninstall-binPROGRAMS uninstall-binSCRIPTS \
 
 mkheader: mkheader.c Makefile
 	$(CC_FOR_BUILD) $(CFLAGS_FOR_BUILD) $(CPPFLAGS_FOR_BUILD) \
-          $(LDFLAGS_FOR_BUILD) -I. -I$(srcdir) -o $@ $(srcdir)/mkheader.c
+          $(LDFLAGS_FOR_BUILD) -I. -I$(srcdir) -o $@ $(srcdir)/mkheader.c $(LIBS)
 
 assuan.h: assuan.h.in mkheader $(parts_of_assuan_h)
 	./mkheader $(host_os) $(srcdir)/assuan.h.in \
